### PR TITLE
Improvements/database migrations

### PR DESCRIPTION
**Pull request by ryszardmadej**
… schema

Issue #11 - adding db migrations.

Note - since we already have tables contained within this migration we will need to drop everything before we run it first time around. And back up the data first if there is a need to preserve it.

### CODE REVIEW

File: migrations/20160525082319-create-db.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+var fs = require('fs');
+var path = require('path');
+var Promise;
+var dotenv = require('dotenv');
+dotenv.config({path: '/Users/ryszardm/work/TechRadar/process.env'});
```
Comment (gpimblott) [inline migrations/20160525082319-create-db.js]: Hardcoded path name to your local directory

Comment (ryszardmadej) [inline migrations/20160525082319-create-db.js]: Ah, idd, did not clean that up

Comment (ryszardmadej) [inline migrations/20160525082319-create-db.js]: removed

**Comment by gpimblott** [comment]
The best set of data is currently held in the 'Production' Heroku instance, which varies siginifically from the boilerplate in the test file.  I suggest we create an initial migration file with no data (even categories) and use the PG backyup from Heroku to populate

**Comment by ryszardmadej** [comment]
Good point, I'll move away all inserts from the migration into the dbtestData.js dataset. We can use that to populate any new test env, and safely restore prod after the migration in that case.

**Comment by ryszardmadej** [comment]
inserts removed from initial migration

**Comment by gpimblott** [comment]
I'll drop the dev/staging databases first;  use the migration script to recreate the DB; then apply the backup.

When that works i''ll do the same to production

**Comment by gpimblott** [comment]
All databases migrated successfully

**Pull request merged**